modules/fedimint-mint-client/src/lib.rs

```diff
@@ -93,10 +93,14 @@ impl FromStr for OOBNotes {
     /// Decode a set of out-of-band e-cash notes from a base64 string.
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         let bytes = base64::decode(s)?;
-        Ok(Decodable::consensus_decode(
+        let oob_notes: OOBNotes = Decodable::consensus_decode(
```

Comment:
Oh, I thought we would do it by making a custom implementation of `Decodable` & `Encodable`. :)